.github/workflows/push_docker_manual.yml

```
@@ -0,0 +1,45 @@
+#### Publish tags to docker hub
+name: 📦 Deploy to Docker with tag
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Enter docker image tag'     
+        required: true
+        default: 'branch'
+
+jobs:
+  deploy:
+    name: 📦 Deploy to Docker with tag
+    runs-on: ubuntu-latest
+    environment: CI - release environment
+    steps:
+      - name: 📤 Checkout the repository 
+        uses: actions/checkout@main
+      - name: 📆 Set version number
+        run: |
+          sed -i '/        private const string Version = /c\        private const string Version =  "dev build";' ${{github.workspace}}/src/DaemonRunner/DaemonRunner/Service/RunnerService.cs
+          sed -i '/    io.hass.version=/c\    io.hass.version="dev" \\' ${{github.workspace}}/Dockerfile
```

Same thing here. Not sure if this works.

```suggestion
          sed -i '/io.hass.version=/c\io.hass.version="dev" \\' ${{github.workspace}}/Dockerfile
```